src/Paysera/Bundle/JavascriptGeneratorBundle/Service/Generator/DateFactoryGenerator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+namespace Paysera\Bundle\JavascriptGeneratorBundle\Service\Generator;
+
+use Paysera\Bundle\CodeGeneratorBundle\Entity\Definition\ApiDefinition;
+use Paysera\Bundle\CodeGeneratorBundle\Entity\SourceCode;
+use Paysera\Bundle\CodeGeneratorBundle\Service\Generator\GeneratorInterface;
+use Symfony\Component\Templating\EngineInterface;
+
+class DateFactoryGenerator implements GeneratorInterface
+{
+    private $twig;
+    private $sourceDir;
+
+    public function __construct(
+        EngineInterface $twig,
+        string $sourceDir
+    ) {
+        $this->twig = $twig;
+        $this->sourceDir = $sourceDir;
+    }
+
+    public function generate(ApiDefinition $definition) : array
+    {
+        $code = $this->twig->render('PayseraJavascriptGeneratorBundle:Package/Src/Service:DateFactory.js.twig');
+
+        return [(new SourceCode())
```

Comment:
Do not think that multiline instance creation inside array is a very good readability